source/client/player/input/ITurnInput.hpp

```diff
@@ -14,7 +14,7 @@ class ITurnInput
 	struct Delta
 	{
 		float x, y;
-		Delta() { x = 0.0f, y = 0.0f; }
+        Delta() { x = 0.0f; y = 0.0f; }
```

Comment:
And this

Reply:
Xcode was giving me warnings about it. The changes I made to it had no affect on the runtime of the program. Does it matter that I changed it?

Reply:
Well it distracts me from the things that you actually changed

Reply:
Would you like me to submit my misc. cleanups in a separate pull-request next time?

Reply:
Sure, please do.